patterns/1-initial/modular-code.md

```diff
@@ -24,33 +24,50 @@ Development does not want to spend the extra resources needed to develop modular
 * Extra documentation is required for reusable code.
 * Some companies have a common components group that develops reusable code, but others feel that such components should be developed by those business lines that are using the components and a library of common components could be established.
 * Developers might not know how to write modular code. Education might be needed.
+
 * Might be a fear that if not done properly, quality might be impacted.
 * Developers might not have incentive to write modular code (due to their tight schedules and lack of a mandate).
 * You might be dealing with legacy systems (can't be simply refactored or rewritten).
 * Requirements might be different for writing modular code.
 * Architectural constraints might impact modularity.
 * Developers who develop monolithic code bases might lack the perspective of how modularity might improve how they work.
+* If there is frequent turnover of team members, modularization may not be a priority.
+* Level of communication between teams can impact ability/inclination to modularize.
 
 ## Solutions
 
 * Provide incentives to teams to invest in modular code. Modular code is far more reusable. This could work well for large teams when working on modularized projects; team members can focus on their smaller assigned tasks.
     - Developers could get an opportunity to increase their influence in the organization.
+    - Modular code makes the pathway to open source smoother.
+    - Modular code is easier to work with, especially for larger teams.
```

Comment:
How about we combine this bullet point and the next by saying: "modular code facilitates division of labour"?

Reply:
I like it.

I added in your suggestion, while steal using part of the original text above as part of the motivation.

```suggestion
    - Modular code facilitates division of labour (as required for work in larger teams, or as an Open Source project)
```